install/scripts/base-ubuntu-18.04.def

```diff
@@ -2,16 +2,18 @@ BootStrap: docker
 From: ubuntu:18.04
 
 %setup
+cp ./install-cmake.sh ${SINGULARITY_ROOTFS}/install-cmake.sh
```

Comment:
*if* we need to enforce cmake 3.16 globally we will also have to install it for the other supported distributions (in particular CentOS) in order to not break the packaging pipeline.

Reply:
Absolutely, can you please point me to the files that need updating and how I can test them? :)

Reply:
no updates for linux needed anymore.